### PR TITLE
Renaming dapps repos. Updating dapps

### PR DESCRIPTION
**Pull request by tomusdrw**
To keep the naming consistent I also renamed all CLI options `webapp` -> `dapps` and all stuff that had `webapp`.

### CODE REVIEW

File: parity/cli.rs [hunk 1]
```diff
@@ -84,19 +84,18 @@ API and Console Options:
   --ipc-apis APIS          Specify custom API set available via JSON-RPC over
                            IPC [default: web3,eth,net,personal,ethcore].
 
-  --webapp-off             Disable the web applications server (e.g. status
-                           page).
-  --webapp-port PORT       Specify the port portion of the WebApps server
+  --dapps-off              Disable the Dapps server (e.g. status page).
+  --dapps-port PORT		   Specify the port portion of the Dapps server
```
Comment (gavofyork) [inline parity/cli.rs]: tabs!

File: parity/cli.rs [hunk 2]
```diff
@@ -168,7 +167,7 @@ Legacy Options:
   --nodekey KEY            Equivalent to --node-key KEY.
   --nodiscover             Equivalent to --no-discovery.
   -j --jsonrpc             Does nothing; JSON-RPC is on by default now.
-  -w --webapp              Does nothing; web app server is on by default now.
+  -d --dapps               Does nothing; dapps server is on by default now.
```
Comment (gavofyork) [inline parity/cli.rs]: don't change the legacy options - they're only there to make old code work and come with a deprecated warning.

**Comment by tomusdrw** [comment]
Fixed.

File: parity/cli.rs [hunk 3]
```diff
@@ -259,7 +258,7 @@ pub struct Args {
 	pub flag_etherbase: Option<String>,
 	pub flag_gasprice: Option<String>,
 	pub flag_jsonrpc: bool,
-	pub flag_webapp: bool,
+	pub flag_dapps: bool,
```
Comment (gavofyork) [inline parity/cli.rs]: `flag_webapp`, still.

Comment (tomusdrw) [inline parity/cli.rs]: Sorry, haven't run the tests :/ Fix on it's way.

**Pull request merged**